core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreWebViewClient.java

```diff
@@ -126,11 +125,7 @@ public void onPageFinished(WebView view, String url) {
       return;
     }
 
-    if (url.equals(HOME_URL)) {
-      onUrlEqualToHome(view);
-    } else {
-      view.removeView(home);
-    }
+    view.removeView(home);
```

Comment:
I am dubious that this `home` variable should even exist. I definitely want to see

`protected void inflateHomeView(WebView view) {`

deleted along with the xml resource `R.layout.content_main`